docs/zz_generated.seed.ee.yaml

```diff
@@ -143,6 +143,9 @@ spec:
           # 'Default' or 'None'. Defaults to "ClusterFirst". DNS parameters given in DNSConfig will be merged with the
           # policy selected with DNSPolicy.
           dnsPolicy: ""
+          # Optional: EnableDefaultNetworkPolicies enables deployment of default network policies like cluster isolation.
+          # Defaults to true.
+          enableDefaultNetworkPolicies: true
```

Comment:
If it's defaulting to true then do you think it makes sense to go with the opposite i.e. `disableDefaultNetworkPolicies`?